src/main/resources/db/migration/V18__Add_chapter_type.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE chapter ADD COLUMN chapter_type TEXT not null DEFAULT 'Content';
```

Comment:
Alltid lurt å ta høgde for at scripta kan kjøres fleire gonger uten feil ved å ta med IF NOT EXISTS i sql.
`ALTER TABLE chapter ADD COLUMN IF NOT EXISTS chapter_type TEXT not null DEFAULT 'Content';`

Reply:
Ja, det er en god tanke å ha med, men ingen av scriptene her via flyway kan kjøres flere ganger dersom de allerede er kjørt en gang. Det er litt av tanken med å bruke Flyway. Den registrerer hver databasemigrering, og vil ikke kjøre en migrering som allerede er kjørt.

Reply:
Eg veit det, men har hatt erfaring med at det likevel har vore nødvendig. Det gjør at du kan slette rad i schema_version og kjøre migreringer på nytt uten at deploy stopper på grunn av sql-feil.